decimalSuite_test.go

```diff
@@ -18,34 +19,62 @@ type testCaseStrings struct {
 	expectedResult string
 }
 
-// TODO(joshcarp): This test cannot fail. Proper assertions will be added once the whole suite passes
-// TestFromSuite is the master tester for the dectest suite.
-// func TestFromSuite(t *testing.T) {
-// 	testVals := getInput("dectest/ddAdd.decTest")
-// 	for _, testVal := range testVals {
-// 		dec64vals := convertToDec64(testVal)
-// 		testErr := runTest(dec64vals, testVal)
-// 		if testErr != nil {
-// 			fmt.Println(testErr)
-// 		}
-// 		if dec64vals.parseError != nil {
-// 			fmt.Println(dec64vals.parseError)
-// 		}
-// 	}
-//
+const TESTDEBUG bool = true
+const RUNSUITES bool = true
+
+var tests = []string{"dectest/ddAdd.decTest"}
+
+// TODO: Implement following tests
+// "dectest/ddCompare.decTest"}
+// 	"dectest/ddMultiply.decTest",
+// 	"dectest/ddAbs.decTest",
+// 	"dectest/ddClass.decTest",
+// 	"dectest/ddCopysign.decTest",
+// 	"dectest/ddDivide.decTest",
+// 	"dectest/ddLogB.decTest",
+// 	"dectest/ddMin.decTest",
+// 	"dectest/ddMinMag.decTest",
+// 	"dectest/ddMinus.decTest",
+// 	"dectest/ddFMA.decTest",
 // }
 
 // TODO: any tests that are failing a particular test in the test suite will be turned into a unit test.
 // func TestNew(t *testing.T) {
-// require := require.New(t)
-// propper rounding
-// require.Equal(MustParseDecimal64("4444444444444445"), MustParseDecimal64("4444444444444444").Add(MustParseDecimal64("0.5001")))
-// require.Equal(MustParseDecimal64("0.23"), MustParseDecimal64("1.3").Add(MustParseDecimal64("-1.07")))
-// fmt.Println("sjoidgf", MustParseDecimal64("4444444444444444").Add(MustParseDecimal64("1.5001")))
-// require.Equal(MustParseDecimal64("12345678901234.29"), MustParseDecimal64("12345678901234").Add(MustParseDecimal64("0.2951")))
-//
+// 	require := require.New(t)
+// 	a := MustParseDecimal64("6543210123456789")
+// 	b := MustParseDecimal64("0.000001")
+// 	c := MustParseDecimal64("6543210123456789")
+// 	ans := b.Add(c)
+// 	// fmt.Println(ansP)
+// 	// g := MustParseDecimal64("sNaN")
+// 	fmt.Println(g.Add(c))
+// 	require.Equal(a, ans)
 // }
 
+// TODO(joshcarp): This test cannot fail. Proper assertions will be added once the whole suite passes
+// TestFromSuite is the master tester for the dectest suite.
+func TestFromSuite(t *testing.T) {
+	if RUNSUITES {
+		for _, i := range tests {
```

Comment:
`i` is normally thought of as an integer. Better to call it `test` (or `file`, after `getInput()`'s parameter).